.github/workflows/test.yml

```diff
@@ -42,6 +42,8 @@ jobs:
       CKAN_DATASTORE_READ_URL: postgresql://datastore_read:pass@postgres/datastore_test
       CKAN_SOLR_URL: http://solr:8983/solr/ckan
       CKAN_REDIS_URL: redis://redis:6379/1
+      # required for CKAN 2.10
+      CKAN__DATAPUSHER__API_TOKEN: XX
```

Comment:
Not if we are not using the datapusher right?

Reply:
Yes, only if datapusher is enabled (it is by default)